src/Concordium.Sdk.csproj

```
@@ -21,6 +21,9 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageLicenseExpression>MPL-2.0</PackageLicenseExpression>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DocumentationFile>bin\Release\net6.0\Concordium.Sdk.xml</DocumentationFile>
+  </PropertyGroup>
```

I think we should use [`GenerateDocumentationFile`](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generatedocumentationfile) instead, and build this for any build, since it will show the warnings related to documentation.
```suggestion
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
  </PropertyGroup>
```